cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -453,11 +484,15 @@ func StaticPodControlPlane(client clientset.Interface, waiter apiclient.Waiter,
 		}
 	}
 
+	// Write the updated static Pod manifests into the temporary directory
```

Comment:
creating the kube-* component man ifests is now required before etcd upgrade wth performEtcdStaticPodUpgrade

Reply:
we'd better to add a some comment about this to prevent bugs caused by accidental code refactoring:
// As part of the design, this must be called before the below `performEtcdStaticPodUpgrade`

Reply:
decided to just add a comment for now.